### ISSUE TITLE
Brainstorming idea

### ISSUE THREAD

**Issue by oren** [JBlocklove/remarkable-daily-pdf#6]
I am curious to hear your thoughts about a script gets URL of a website, convert it to PDF and upload to the rm.

I have a bash script that I use to upload pdf to my rm:

./upload file.pdf

```
curl --form file="@$1" http://10.11.99.1/upload
```

If we add a conversion of a website to PDF we can use this script instead of installing things on the rm itself.

**Comment by JBlocklove** [comment]
That's a bit out of scope of the project. The whole point of this is to give this functionality directly on the remarkable, rather than needing to communicate with a computer. If I end up implementing any type of website to PDF conversion it's pretty much just going to be a print to PDF sort of thing.

Realistically, I made this specifically to pull a pdf that updates every day like my daily crossword from the NYT. If you end up having specific one-off sites that you want to sync you'd have to ssh into the tablet anyway and that kind of ends up defeating the point. I may make another repo that allows you to push files from a computer using the functions I made for this, but that has some of its own challenges so I haven't really touched it yet.
